shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -1136,6 +1140,12 @@ public void attachToFlutterEngine(@NonNull FlutterEngine flutterEngine) {
       Log.e(TAG, "TextServicesManager not supported by device, spell check disabled.");
     }
 
+    if (Build.VERSION.SDK_INT >= API_LEVELS.API_34) {
```

Comment:
I am trying to work out the safety of annotating the plugin with requires 34 and using this code to protect against bad calls. My gut says the plugin should always be created and it should handle the default behavior when running on something lower than 33 but I cant find that in any of our documentation or style guide. 

Additionally the channel is being created in a bunch of code and used in a bunch of code without the api protections. 

Are you confidant that the api methods cannot be called through the channel if we never create the plugin?

Reply:
For simplicity we can say that flutter supports the feature on api 34 and above.

Reply:
I've removed this `if` so that ScribePlugin is still instantiated on older SDK versions, but the methods themselves are locked down to the correct API level. I also added tests for unsupported API levels as @reidbaker mentioned in another comment.